flink-connectors/flink-connector-hive/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,41 @@
+flink-connector-hive
+Copyright 2014-2019 The Apache Software Foundation
+
```

Comment:
some dependencies are missing from the list:
```
org.apache.commons:commons-lang3:3.3.2
org.apache.commons:commons-compress:1.18
org.apache.hive:hive-exec:2.3.4
```

Reply:
I included hive-exec. Also updated `<artifactSet>` to include only dependencies we explicitly need, so libs included in hive-exec but not declared as dependency, like commons-lang3 and commons-compress, can be excluded from flink-connector-hive jar.